fhir_2_ada-r4/lab/3.0.0/sturen_laboratoriumresultaten/payload/sturen_laboratoriumresultaten_2_ada.xsl

```diff
@@ -74,7 +77,10 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
 								<xsl:copy-of select="(f:Bundle/f:entry/f:resource/f:Organization)[1]/ancestor::f:entry"/>
 							</xsl:when>
 							<xsl:otherwise>
-								<xsl:message terminate="yes">FATAL: Unable to determine custodian (beschikbaarstellende_partij) from incoming data. </xsl:message>
+								<xsl:call-template name="util:logMessage">
+									<xsl:with-param name="level" select="$logFATAL"/>
+									<xsl:with-param name="msg">Unable to determine custodian (beschikbaarstellende_partij) from incoming data. The custodian is determined based on MedicationRequest/requester, MedicationDispense/performer, or MedicationStatement.extension[author] and constitutes a required piece in the target transaction</xsl:with-param>
+								</xsl:call-template>
```

Comment:
Add `<xsl:with-param name="terminate" select="true()"/>`. Also for other FATAL messages below.

Reply:
Ja, of als het niet fatal is, dan een $logError. Zelf ben ik van de school garbage in / garbage out. Daarmee leg je het probleem buiten neer.

Reply:
I've applied the terminate = true() param as was done across the board, to be consistent